src/jinja2/filters.py

```diff
@@ -524,15 +540,15 @@ def do_last(environment, seq):
 
 
 @contextfilter
-def do_random(context, seq):
+def do_random(context, seq: Sequence) -> Any:
```

Comment:
I guess I am not clear when to use a TypeVar of T that represents an Any generic type vs just using `Any` and/or a group type like Sequence that represents any type of sequence types. If by `Sequence[T]` could we also just say `Sequence[Any]` instead of using generic type T?